docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md

```diff
@@ -41,45 +41,52 @@ Complete the steps in one of the following sections to set up an add-in catalog
 
 9. To add an add-in to the Office Add-ins add-in catalog, choose **Apps for Office**.
 
-### To set up an add-in catalog on Office 365
+### To set up an app catalog on Office 365

```

Comment:
This article uses the term "set up" in many places. Seems like this phrase may be bad for localization (i.e., may not translate accurately to other languages)...do you agree?  If so, then recommend that you replace all instances of "set up" with another phrase like "create" or "create and configure" or whatever verb(s) accurately describe what's being done.

Reply:
fixed